src/actions/index.js

```diff
@@ -24,6 +24,6 @@ export function addCat() {
                 }).catch(() => {
                     dispatch(endDowloadingImages())
                 });
-        }
+        } else return Promise.resolve();
```

Comment:
_minor_: it's not a good practice to modify the production code for the sake of the tests. let's remove this and use async/await in tests instead.